contrib/builder-keys/README.md

```diff
@@ -20,7 +20,7 @@ To fetch keys of builders and active developers, feed the list of fingerprints
 of the primary keys into gpg:
 
 ```sh
-while read fingerprint keyholder_name; do gpg --keyserver hkp://subset.pool.sks-keyservers.net --recv-keys ${fingerprint}; done < ./keys.txt
+while read fingerprint keyholder_name; do gpg --keyserver hkps://keys.openpgp.org --recv-keys ${fingerprint}; done < ./keys.txt
```

Comment:
Are all builder keys on that keyserver? IIRC only persons with access to the email that the key is attached to can upload there.

Also the `src/minisketch/test` file added in this pull seems unrelated.

Reply:
Nvm, just read your previous comment:

> Looks like keys.opengpg.org also has more of the builder keys available than the Ubuntu server, as well as some newer signatures for keys that are available on either.